src/mono/System.Private.CoreLib/src/System/TypeLoadException.Mono.cs

```diff
@@ -13,6 +13,18 @@ internal TypeLoadException(string className, string assemblyName)
             _assemblyName = assemblyName;
```

Comment:
Can me make the `string className, string assemblyName` constructor private, so that it is hard to call it by mistake from random places? (If it is private, it can be only called from within the TypeLoadException implementation.)

Reply:
Done, and suppressed the suggestion to remove the constructor.